docs/manifest.json

```diff
@@ -49,6 +49,12 @@
         }
       ]
     },
+    {
+      "title": "Collaboration",
+      "description": "How to setup real time collaboration using code server.",
+      "icon": "<svg viewBox=\"0 0 16 16\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M4 6H2v14c0 1.1.9 2 2 2h14v-2H4V6zm16-4H8c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 10l-2.5-1.5L15 12V4h5v8z\"></path></svg>",
```

Comment:
This is using the same icon from the `Usage` section. I did not know from which icon library to get a new one, will need some help with that

Reply:
cc @jsjoeio @BrunoQuaresma

Reply:
@BrunoQuaresma can you let us know which icon we should use for Collaboration and help @galesky out here?

Reply:
Here is the SVG
```html
<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M12.2 13.4357L9.5 11.4357C10.4 10.7357 11 9.7357 11 8.5357V7.7357C11 5.8357 9.6 4.1357 7.7 4.0357C5.7 3.9357 4 5.5357 4 7.5357V8.5357C4 9.7357 4.6 10.7357 5.5 11.4357L2.8 13.5357C2.3 13.9357 2 14.5357 2 15.1357V17.0357C2 17.6357 2.4 18.0357 3 18.0357H12C12.6 18.0357 13 17.6357 13 17.0357V15.0357C13 14.4357 12.7 13.8357 12.2 13.4357Z" fill="black"/>
<path d="M17.1 8.43436L15.3 7.23436C15.7 6.83436 16 6.23436 16 5.53436V4.63436C16 3.43436 15.1 2.23436 13.9 2.03436C12.7 1.83436 11.7 2.53436 11.2 3.43436C12.3 4.43436 13 5.83436 13 7.43436V8.43436C13 9.33436 12.8 10.2344 12.4 10.9344C12.4 10.9344 13.6 11.8344 13.6 11.9344H17C17.6 11.9344 18 11.5344 18 10.9344V10.1344C18 9.43436 17.7 8.83436 17.1 8.43436Z" fill="black"/>
</svg>
```

Here is the icon:
<img width="70" alt="Screen Shot 2021-09-13 at 15 44 00" src="https://user-images.githubusercontent.com/3165839/133139337-440241c7-b310-47c8-97fd-ddfce3045c20.png">

Reply:
Amazing! Thanks @BrunoQuaresma 

@galesky do you mind using this SVG instead? Then you can @ me once it's changed and we can get this merged

Reply:
Thanks, @BrunoQuaresma!

I just committed the new SVG, I think we are ready to merge

It's a pity that the 'docs preview pipeline' is failing :(

Reply:
Awesome! 

I know, we're super sorry about that :( We just added it a couple weeks ago and didn't anticipate this issue. I added it to the next milestone so it's being prioritized: https://github.com/cdr/code-server/issues/4027

I'll merge this! 

Thank you again! ♥️